docs/src/main/asciidoc/quartz.adoc

```diff
@@ -400,6 +400,20 @@ After a few seconds, in another terminal, run `curl localhost:8080/tasks` to ver
 
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+[[quartz-register-plugin-listeners]]
+== Registering Plugin and Listeners
+
+You can register a plugin, jobListener or triggerListener through Quarkus configuration.
+
+The example bellow register the plugin `org.quartz.plugins.history.LoggingJobHistoryPlugin` named as `jobHistory` with the property `jobSuccessMessage` defined as `Job [{1}.{0}] execution complete and reports: {8}` 
+
+[source,conf]
+----
+quarkus.quartz.plugin.jobHistory.class=org.quartz.plugins.history.LoggingJobHistoryPlugin
+quarkus.quartz.plugin.jobHistory.jobSuccessMessage=Job [{1}.{0}] execution complete and reports: {8}
+----
+
+
```

Comment:
This is much better. Thank you. 

 Also in this issue we wanted to document the programmatic way of doing things as pointed out by @mkouba in https://github.com/quarkusio/quarkus/issues/10962#issuecomment-664152333. Can you add a note in the documentation about that? Thank you.

Reply:
I included this information on the documentation. But I didn't find on Quartz API where to register a Plugin programmatic. So I just pointed out this way for jobListener or triggerListener.

Do you know if it is possible?

Reply:
Nice work. I do not know if it possible.